src/receive_imf.rs

```diff
@@ -1443,61 +1442,27 @@ async fn add_parts(
 
     if let Some(node_addr) = mime_parser.get_header(HeaderDef::IrohNodeAddr) {
         chat_id = DC_CHAT_ID_TRASH;
```

Comment:
you could already short circuit here and not look up mid from the db but its fine how it is